
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />

    <title>Chapter 3 - Hallmarks and agng clocks &#8212; Computational Longevity</title>
    
  <!-- Loaded before other Sphinx assets -->
  <link href="_static/styles/theme.css?digest=1999514e3f237ded88cf" rel="stylesheet">
<link href="_static/styles/pydata-sphinx-theme.css?digest=1999514e3f237ded88cf" rel="stylesheet">

    
  <link rel="stylesheet"
    href="_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    <link rel="stylesheet" type="text/css" href="_static/pygments.css" />
    <link rel="stylesheet" href="_static/styles/sphinx-book-theme.css?digest=5115cc725059bd94278eecd172e13a965bf8f5a9" type="text/css" />
    <link rel="stylesheet" type="text/css" href="_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="_static/mystnb.css" />
    <link rel="stylesheet" type="text/css" href="_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="_static/design-style.b7bb847fb20b106c3d81b95245e65545.min.css" />
    
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="_static/scripts/pydata-sphinx-theme.js?digest=1999514e3f237ded88cf">

    <script data-url_root="./" id="documentation_options" src="_static/documentation_options.js"></script>
    <script src="_static/jquery.js"></script>
    <script src="_static/underscore.js"></script>
    <script src="_static/doctools.js"></script>
    <script src="_static/clipboard.min.js"></script>
    <script src="_static/copybutton.js"></script>
    <script src="_static/scripts/sphinx-book-theme.js?digest=9c920249402e914e316237a7dbc6769907cce411"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="_static/togglebutton.js"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown, .tag_hide_input div.cell_input, .tag_hide-input div.cell_input, .tag_hide_output div.cell_output, .tag_hide-output div.cell_output, .tag_hide_cell.cell, .tag_hide-cell.cell';</script>
    <script src="_static/design-tabs.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="_static/sphinx-thebe.js"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Chapter 4 - Longevity interventions and models" href="Chapter%204%20-%20Longevity%20interventions%20and%20models.html" />
    <link rel="prev" title="Chapter 2 - Biology basics and measuring biology" href="Chapter%202%20-%20Biology%20basics%20and%20measuring%20biology.html" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="docsearch:language" content="None">
    

    <!-- Google Analytics -->
    
  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="60">
<!-- Checkboxes to toggle the left sidebar -->
<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">
<label class="overlay overlay-navbar" for="__navigation">
    <div class="visually-hidden">Toggle navigation sidebar</div>
</label>
<!-- Checkboxes to toggle the in-page toc -->
<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">
<label class="overlay overlay-pagetoc" for="__page-toc">
    <div class="visually-hidden">Toggle in-page Table of Contents</div>
</label>
<!-- Headers at the top -->
<div class="announcement header-item noprint"></div>
<div class="header header-item noprint"></div>

    
    <div class="container-fluid" id="banner"></div>

    

    <div class="container-xl">
      <div class="row">
          
<!-- Sidebar -->
<div class="bd-sidebar noprint" id="site-navigation">
    <div class="bd-sidebar__content">
        <div class="bd-sidebar__top"><div class="navbar-brand-box">
    <a class="navbar-brand text-wrap" href="index.html">
      
      
      
      <h1 class="site-logo" id="site-title">Computational Longevity</h1>
      
    </a>
</div><form class="bd-search d-flex align-items-center" action="search.html" method="get">
  <i class="icon fas fa-search"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search this book..." aria-label="Search this book..." autocomplete="off" >
</form><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="Chapter%201%20-%20Motivation.html">
                    Contents of this book:
                </a>
            </li>
        </ul>
        <ul class="current nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="Chapter%202%20-%20Biology%20basics%20and%20measuring%20biology.html">
   Chapter 2 - Biology basics and measuring biology
  </a>
 </li>
 <li class="toctree-l1 current active">
  <a class="current reference internal" href="#">
   Chapter 3 - Hallmarks and agng clocks
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="Chapter%204%20-%20Longevity%20interventions%20and%20models.html">
   Chapter 4 - Longevity interventions and models
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="Chapter%205%20-%20References.html">
   References
  </a>
 </li>
</ul>

    </div>
</nav></div>
        <div class="bd-sidebar__bottom">
             <!-- To handle the deprecated key -->
            
            <div class="navbar_extra_footer">
            Powered by <a href="https://jupyterbook.org">Jupyter Book</a>
            </div>
            
        </div>
    </div>
    <div id="rtd-footer-container"></div>
</div>


          


          
<!-- A tiny helper pixel to detect if we've scrolled -->
<div class="sbt-scroll-pixel-helper"></div>
<!-- Main content -->
<div class="col py-0 content-container">
    
    <div class="header-article row sticky-top noprint">
        



<div class="col py-1 d-flex header-article-main">
    <div class="header-article__left">
        
        <label for="__navigation"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="right"
title="Toggle navigation"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-bars"></i>
  </span>

</label>

        
    </div>
    <div class="header-article__right">
<button onclick="toggleFullScreen()"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="bottom"
title="Fullscreen mode"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>

<div class="menu-dropdown menu-dropdown-download-buttons">
  <button class="headerbtn menu-dropdown__trigger"
      aria-label="Download this page">
      <i class="fas fa-download"></i>
  </button>
  <div class="menu-dropdown__content">
    <ul>
      <li>
        <a href="_sources/Chapter 3 - Hallmarks and aging clocks.md"
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Download source file"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="headerbtn__text-container">.md</span>
</a>

      </li>
      
      <li>
        
<button onclick="printPdf(this)"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="left"
title="Print to PDF"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="headerbtn__text-container">.pdf</span>
</button>

      </li>
      
    </ul>
  </div>
</div>
<label for="__page-toc"
  class="headerbtn headerbtn-page-toc"
  
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-list"></i>
  </span>

</label>

    </div>
</div>

<!-- Table of contents -->
<div class="col-md-3 bd-toc show noprint">
    <div class="tocsection onthispage pt-5 pb-3">
        <i class="fas fa-list"></i> Contents
    </div>
    <nav id="bd-toc-nav" aria-label="Page">
        <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#new-hallmarks-of-aging">
   New hallmarks of aging
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#extracellular-matrix-ecm-and-its-connection-to-longevity">
     Extracellular matrix (ECM) and its connection to longevity
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#new-hallmarks-of-ageing-a-2022-copenhagen-ageing-meeting-summary">
   New hallmarks of ageing: a 2022 Copenhagen ageing meeting summary
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#autophagy">
     1. Autophagy
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#inflammation">
     2. Inflammation
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#aging-clocks-causality-and-the-future-of-evaluating-biological-age">
   Aging clocks, causality and the future of evaluating biological age.
  </a>
 </li>
</ul>

    </nav>
</div>
    </div>
    <div class="article row">
        <div class="col pl-md-3 pl-lg-5 content-container">
            <!-- Table of contents that is only displayed when printing the page -->
            <div id="jb-print-docs-body" class="onlyprint">
                <h1>Chapter 3 - Hallmarks and agng clocks</h1>
                <!-- Table of contents -->
                <div id="print-main-content">
                    <div id="jb-print-toc">
                        
                        <div>
                            <h2> Contents </h2>
                        </div>
                        <nav aria-label="Page">
                            <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#new-hallmarks-of-aging">
   New hallmarks of aging
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#extracellular-matrix-ecm-and-its-connection-to-longevity">
     Extracellular matrix (ECM) and its connection to longevity
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#new-hallmarks-of-ageing-a-2022-copenhagen-ageing-meeting-summary">
   New hallmarks of ageing: a 2022 Copenhagen ageing meeting summary
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#autophagy">
     1. Autophagy
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#inflammation">
     2. Inflammation
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#aging-clocks-causality-and-the-future-of-evaluating-biological-age">
   Aging clocks, causality and the future of evaluating biological age.
  </a>
 </li>
</ul>

                        </nav>
                    </div>
                </div>
            </div>
            <main id="main-content" role="main">
                
              <div>
                
  <section class="tex2jax_ignore mathjax_ignore" id="chapter-3-hallmarks-and-agng-clocks">
<h1>Chapter 3 - Hallmarks and agng clocks<a class="headerlink" href="#chapter-3-hallmarks-and-agng-clocks" title="Permalink to this headline">#</a></h1>
<p>Contents of this chapter are:</p>
<ul class="simple">
<li><p>Hallmarks of aging and how tackle them computationally.</p></li>
<li><p>New hallmarks of aging and how tackle them computationally.</p></li>
<li><p>Aging clocks, causality and the future of evaluating biological age.</p></li>
</ul>
<p><img alt="" src="https://nintil.com/images/2020-01-05-longevity/image-20200117094655213.png" /></p>
<p>Inspired by the seminal paper in cancer research, the <a class="reference external" href="https://www.cell.com/fulltext/S0092-8674(11)00127-9">hallmarks of cancer</a>, there has been a similiar attempt for aging.</p>
<p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/23746838/">Seminal paper</a> defined the nine (then extended to 10) main “causes” of our decline, the ten horsemen of an internal apocalypse. The question I wanted to answer is:</p>
<blockquote>
<div><p><em>Can we frame these problems/horsemen computationally in terms of machine learning and then try to solve them?</em></p>
</div></blockquote>
<ol class="simple">
<li><p><strong>Genomic instability</strong>: DNA doesn’t always replicate according to plan. Typically, these errors in gene expression get caught and corrected, but not always. Over time, these misfires build up, causing our body to wear down — meaning genetic instability leads to genetic damage leads to a limit on lifespan. Think of it as a broken copy machine, except, instead of producing unreadable pages, our broken genetic copier produces diseases like cancer, muscular dystrophy and ALS. <strong>Data?</strong> <a class="reference external" href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5330-5">HumCFS</a>: a database of fragile sites in human chromosomes. <strong>What could you do with it?</strong> Identify other fragile chromosomes sites AND link them to other diseases based on atom/chemical level properties.</p></li>
<li><p><strong>Telomere Attrition</strong>: At the heart of a cell, DNA is packed into threadlike structures called chromosomes. Chromosomes are capped by telomeres which act like barriers — like bumpers on a car — designed to protect the core of the chromosome. But as DNA replicates, telomeres get shorter and at a critical level, the cell stops dividing, and we become more prone to disease and death. It has been shown that if one can increase or reduce the shrinkage of telomere on mice, longevity increases. <strong>Data?</strong> <a class="reference external" href="http://telomerase.asu.edu/">Telomerase Database</a> <strong>What could you do with it?</strong> <a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/28218725/">Telomerase can be used as a target in anti-cancer therapy</a>. What this essentially means is one can find computationally (in-silico) inhibitors that hinder the shrinkage of telomerase. Doing this which we know has positive side effects for longevity.</p></li>
<li><p><strong>Epigenetic Alterations</strong>: Nature impacts nurture. Over the course of a lifetime, environmental toxins can change how our genes express, sometimes for the worst. Exposure to carcinogens in the environment can silence the gene that suppresses tumors for example. But also the other way around, we can silence the gene that is responsible for cancer. <strong>Data?</strong> <a class="reference external" href="https://www.nature.com/articles/srep19340">dbEM</a> and <a class="reference external" href="https://genomics.senescence.info/genes/index.html">GenAge</a>: A database of epigenetic modifiers curated from cancerous and normal genomes. <strong>What could you do with it?</strong> Epigenetic proteins (modifiers) are targets in cancer. With this database we get not only the protein but also genomic information pertaining to mutational status, copy number variation and expression level of epigenetic proteins in tumor samples. This enables explorationof other protein that were not classified as epigenetic, based upon their genomic information. <strong>How to utilise GenAge?</strong> There are tools and companies out there that allow for a complete measurement of one genes. ML models could be built that recommend actions based on certain gene structure.</p></li>
<li><p><strong>Loss of Proteostasis</strong>: Inside a cell, proteins run the show. They transport materials, send signals, switch processes on and off, and provide structural support. But proteins become less effective over time, so the body recycles them. Unfortunately, as we age, we can lose this ability. The trash collector goes on strike and we suffer a toxic buildup of proteins that can, for example, lead to diseases such as Alzheimer’s. <strong>Data?</strong> <a class="reference external" href="https://alphafold.ebi.ac.uk/">AlphaFold</a>: Protein Structure Database. <strong>What could you do with it?</strong> Proteins that become terminally misfolded, or are no longer functionally required, they must be degraded to avoid damaging effects of their continued presence, predicting these proteins for shapes that we do not know could be essential in targeting their degradation. Which is exactly what we have achieved with Celeris (using a different approach)</p></li>
<li><p><strong>Nutrient Sensing Goes Awry</strong>: The human body relies on over forty different nutrients to stay healthy. For everything to work perfectly, cells need to be able to recognize and process each of these. But this ability breaks down as we get older. For example, one reason people gain weight as they age is that our cells can no longer properly digest fat and carbohydrates. And one reason we die is that this impacts the insulin and IGF-1 pathway and can result in diabetes. <strong>Data?</strong> <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4313349/">Nutrient Sensing Mechanisms and Pathways</a>. <strong>What could you do with it?</strong> <a class="reference external" href="https://genomics.senescence.info/diet/">GenDR Database of Dietary Restriction-Related Genes</a>. Not a direct ML approach but utilising <a class="reference external" href="https://pubs.acs.org/doi/10.1021/acssensors.1c00553">advancements in wearables and mobile sensors</a> to get a more granular and personalised data on nutrition intake based on your current metabolic state could have huge impact. One can easily imagine getting measuraments on nutrient sensing of different (cancer) cells in the body and then depending on that information adjusting the nutrient intake such that the desired effect on cells is achieved. For example death of cancer cells. Extending this to genes one can use the GenDR Database of Dietary Restriction-Related Genes</p></li>
<li><p><strong>Mitochondrial Dysfunction</strong>: Mitochondria are power plants. By converting oxygen and food into energy, they provide the basic fuel for our cells. But performance declines over time. The result is free radicals, a damaging form of oxygen that mangles DNA and proteins and leads to many of the chronic illnesses associated with aging. <strong>Data?</strong> <a class="reference external" href="https://mseqdr.org/mitobox.php">MSeqDR</a> Mitochondrial Genomic Tools and Databases <strong>What could you do with it?</strong> With this currated list of databses one could combine measurements of Oxygen consumption ratio (OCR), maximal oxygen consumption and mitochondrial reserve capacity and their respective proteins. Beeing especially interested <a class="reference external" href="https://academic.oup.com/nar/article/47/D1/D1225/5162470?login=false">in proteins that are targets in certain diseases</a>, one could then have clear indicator with wearables or other measurement devices wether some disease is going to be triggered.</p></li>
<li><p><strong>Cellular Senescence</strong>: As cells undergo stress, they occasionally become “senescent”, both losing their ability to divide and becoming resistant to death. These “zombie cells” can’t be removed from the body. They build up over time, infect neighboring cells, and ultimately create a zombie apocalypse of inflammatory debilitation. <strong>Data?</strong> <a class="reference external" href="https://genomics.senescence.info/cells/">CellAge</a> Database of Cell Senescence Genes <strong>What could you do with it?</strong> Based on our personal gene makeup we could have model that recommend altering of genes associated with cellular senescence. Instead of altering <a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/32264951/">one can do further analysis of what action</a> is needed for the genes involved in the sensescence.</p></li>
<li><p><strong>Stem Cell Exhaustion</strong>: As we age, our supply of stem cells plummets, in certain cases by a ten thousand fold decline. Worse, the ones we do manage to hang on to become far less active. This means that the body’s internal tissue and organ repair system loses its ability to do its job. Now we already know that stem cells help and there are therapies out there for them. But how can AI make this even better? <strong>Data?</strong> <a class="reference external" href="https://academic.oup.com/nar/advance-articles?login=false">Stemformatics</a>: visualize and download curated stem cell data <strong>What could you do with it?</strong> A curated list of potenital AI applications in stem cell research can be found <a class="reference external" href="https://www.eurekaselect.com/article/110522">here</a>. One crucial application in designing stem cell therapies for central nervous system (CNS) diseases is differentiation of neural stem cells (NSCs) into neurons. Differentiation is complex and not yet clearly established, especially at the early stage. Novel <a class="reference external" href="https://www.nature.com/articles/s41467-021-22758-0">nature</a> publication shows that deep learning could extract minutiae from large-scale datasets, and present a deep neural network model for predictable reliable identification of NSCs fate.</p></li>
<li><p><strong>Altered Inter Cellular Communication</strong>: For the body to function properly, cells need to communicate. This happens constantly, with messages flowing through our bloodstream, immune system and endocrine system. Over time, signals get crossed. Some cells become unresponsive, others become inflammation-producing zombie cells. This inflammation blocks further communication. Once this happens, messages can’t get through and the immune system can’t find pathogens. <strong>Data?</strong> <a class="reference external" href="https://panglaodb.se/">PanglaoDB</a>: is a database for the scientific community interested in exploration of single cell RNA sequencing experiments from mouse and human. <strong>What could you do with it?</strong> Developing a ML tool that can quantitatively infer and analyze intercellular communication would be useful because increases and decreses in signal between cells could be indicative of changes in the organism and other action should be taken. Novel nature publication develops such tool. <a class="reference external" href="https://www.nature.com/articles/s41467-021-21246-9">CellChats</a>.</p></li>
<li><p><strong>Protein crosslinking</strong>: This is a phenomenon where multiple separate proteins become bonded together by a sugar molecule in a process called glycation. Depending upon where this occurs, it tends to correlate with different signs of aging — wrinkles, arteriosclerosis, cataracts, and kidney failure, to name a few.</p></li>
</ol>
<p><strong>Data?</strong> <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4977572/">ProXL</a> (Protein Cross-Linking Database): A Platform for Analysis, Visualization, and Sharing Protein Cross-Linking Mass Spectrometry Data. <strong>What could you do with it?</strong> Developing an ML to identify where this bondage/linkage occurs and what kind of damage it will cause. If labeled data is hard to obtain, clustering techniques can be used to analyze already known bonding regions and try to abstract from there.</p>
<blockquote>
<div><p><strong>But they are not enough</strong></p>
</div></blockquote>
<p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/34271186/">JPM argues</a>, that the proposed hallmarks are not causative and explanatory enough of the true complexity of the underlying processes.</p>
<blockquote>
<div><p><em>“In this context, the scheme is
akin to a folding screen bearing a hallmarks of aging diagram that blocks
the view of the true state of undress of the field. Shivering behind the
screen is the ailing damage maintenance paradigm. We argue that as a
field we must look and move beyond the hallmarks to understand the
process of aging. Only by doing so can paradigms be formulated that
possess sufficient explanatory power to enable treatments for human
aging to be developed.”</em></p>
</div></blockquote>
<p>It comes back to one of the fallacy I outlined: <strong>Oversimplification of biology.</strong></p>
<p>So what can be done? Taking the table underneath as insight, one could quantify every possible biomarker. Determine which are statistically significant and try to map novel theory (hallmark) that could causative explanation of this multifactorial complex process we call aging.</p>
<p>The American Federation for Aging Research (AFAR) recommends the following criteria for biomarkers of aging:</p>
<ol class="simple">
<li><p>It must predict a person’s physiological, cognitive, and physical function in an age-related way, independently of chronological age.</p></li>
<li><p>It must be testable and not harmful to test subjects (for example a blood test or an imaging technique); it must also be technically simple to perform, and it must be accurate and reproducibly without the need for specialized equipment or techniques.</p></li>
<li><p>It should work in laboratory animals as well as humans since preliminary testing is always done in nonhuman subjects.</p></li>
</ol>
<p>Check <a class="reference external" href="https://link.springer.com/article/10.1007/s11912-020-00977-w/tables/2">here</a> for additional details:</p>
<p><img alt="" src="https://i.postimg.cc/Yq4CNqVz/hallmark.jpg" /></p>
<p>Another important reference for the biomarkers of aging is the <a class="reference external" href="https://www.frontiersin.org/articles/10.3389/fgene.2021.686320/full">following</a>. The rc-score displays how often a potential biomarker was referred to in reviews, based on the above-described NCBI-PubMed queries.</p>
<p>Then effort score was added for each potential biomarker. Following values were considered:</p>
<p>• A low e-score (-) describes a potential biomarker which is easy to sample, to handle and to process (usually automated fast and reliable measurements are available from known sources). For example, blood counts from venous blood qualify, since such blood can be sampled easily, the plasma/serum can be stored at room temperature (up to 3 h for many analytes) or in a standard freezer (≥3 month for most analytes) and be processed by equipment that is regularly available in a standard diagnostic/clinical unit or research laboratory. The costs are low to moderate (≤10 €).</p>
<p>• A moderate e-score (–) is assigned if one step (sampling, handling, or processing) is associated with substantial extra effort for routine laboratories. This includes sampling under special conditions, a requirement for prompt sample handling, or the need for elaborate validation.</p>
<p>• A high e-score (—) implies elaborate sampling (e.g., biopsy, lumbar puncture, etc.), handling (e.g., storage in liquid nitrogen) and/or processing (e.g., non-routine nucleotide or protein sequencing). The financial costs are usually high.</p>
<p><img alt="" src="https://www.frontiersin.org/files/Articles/686320/fgene-12-686320-HTML/image_m/fgene-12-686320-t001.jpg" /></p>
<p>Again the relevance of these biomarkers is that they could be utilized not only to elucidate new theories and hallmarks computationally but also construct <a class="reference external" href="https://www.technologyreview.com/2022/04/15/1050019/aging-clocks/">aging clocks</a>.</p>
<section id="new-hallmarks-of-aging">
<h2>New hallmarks of aging<a class="headerlink" href="#new-hallmarks-of-aging" title="Permalink to this headline">#</a></h2>
<p>As the research progresses new theories started to emerge that aim to extend the current hallmarks.</p>
<section id="extracellular-matrix-ecm-and-its-connection-to-longevity">
<h3>Extracellular matrix (ECM) and its connection to longevity<a class="headerlink" href="#extracellular-matrix-ecm-and-its-connection-to-longevity" title="Permalink to this headline">#</a></h3>
<p>The extracellular matrix (ECM) is a network of proteins and other molecules that support, surround, and give structure to cells and tissues in the body. The ECM helps cells attach to and communicate with nearby cells, and plays an important role in cell growth, movement, and other functions. This three-dimensional network provides physical structure to the body and serves as a substrate for cells to live and grow. Cells often modify the components of the ECM, such as osteoclasts and osteoblasts, which degrade and lay down bone ECM, respectively.</p>
<p>Changes to the ECM are often overlooked as a cause of aging, but they can drastically impact cell behavior. Studies have shown that ECM characteristics such as stiffness and chemical composition can affect cell behaviors including proliferation rate, secretome, differentiation, and cellular maintenance. There is growing consensus that ECM-related changes, particularly matrix stiffening, should be considered on a similar level to the other hallmarks of aging.</p>
<p>“Accumulation of damage to the ECM drives cellular aging and disease progression. Collagen fragmentation, collagen glycation, collagen crosslinking, and aggregation of proteins in the extracellular space affect ECM integrity to accelerate agin.” From <a class="reference external" href="https://www.karger.com/Article/FullText/504295">The Matrisome during Aging and Longevity</a></p>
<p><img alt="" src="https://www.researchgate.net/profile/Abdelaziz-Ghanemi/publication/340352699/figure/fig1/AS:875550012895237&#64;1585758798873/Example-of-extracellular-matrix-ECM-implications-The-extracellular-matrix-is-involved.png" /></p>
</section>
</section>
<section id="new-hallmarks-of-ageing-a-2022-copenhagen-ageing-meeting-summary">
<h2>New hallmarks of ageing: a 2022 Copenhagen ageing meeting summary<a class="headerlink" href="#new-hallmarks-of-ageing-a-2022-copenhagen-ageing-meeting-summary" title="Permalink to this headline">#</a></h2>
<p>After almost a decade, we have gotten a <a class="reference external" href="https://www.aging-us.com/article/204248/text">formal update</a> on potential novel hallmarks of aging. Here is the list:</p>
<ol class="simple">
<li><p>Autophagy</p></li>
<li><p>Inflammation</p></li>
<li><p>Microbiome disturbance</p></li>
<li><p>Altered mechanical properties</p></li>
<li><p>Splicing dysregulation</p></li>
</ol>
<p><img alt="" src="https://www.aging-us.com/article/204248/figure/f1/hero" /></p>
<p><em>Now our goal is same as for the original hallmarks. We want to describe them briefly, comment the data for them and then pose some question that can be answered computationally.</em></p>
<section id="autophagy">
<h3>1. Autophagy<a class="headerlink" href="#autophagy" title="Permalink to this headline">#</a></h3>
<p><img alt="" src="https://images.theconversation.com/files/436311/original/file-20211208-188518-1fyp5pw.png?ixlib=rb-1.1.0&amp;q=45&amp;auto=format&amp;w=754&amp;h=448&amp;fit=crop&amp;dpr=1" /></p>
<p>Senescence and autophagy are essential cellular processes that play a role in aging. Senescent cells are damaged cells that stop dividing and secrete cytokines and chemokines that can cause inflammation in neighboring cells. This can lead to tissue damage and promote cancer. Autophagy, on the other hand, is the process by which cells break down and recycle damaged components. This can help repair and prevent damage to cells. Interventions that target pathways involved in aging often upregulate autophagy. Some drugs can also promote autophagy, which can help alleviate the hallmarks of aging.</p>
<p><strong>What are some interesting computational research questions concerning autophagy?</strong></p>
<p>If we accept that the molecular damage is confounding variable in aging process, then getting rid of damaged objects would aid in tackling the problem.</p>
<p><strong>Autophagy measuring and prediction through biomarkers:</strong></p>
<ul class="simple">
<li><p>Tsuboyama, Kotaro, et al. “The ATG conjugation systems are important for degradation of the inner autophagosomal membrane.„ - <strong>gives evidence of a direct correlation between LC3-II and p62 and autophagy level</strong></p></li>
<li><p>Bobillo Lobato, Joaquin, Maria Jiménez Hidalgo, and Luis M. Jiménez Jiménez. “Biomarkers in lysosomal storage diseases.“ — <strong>links potential biomarkers.</strong></p></li>
</ul>
<p>Both together give the ability to model the autophagy (flux) levels insilico.</p>
<p>So the approach would look something like this:</p>
<p><a href="https://ibb.co/h20my03"><img src="https://i.ibb.co/6DQWmQj/autophagy.jpg" alt="autophagy" border="0"></a></p>
<p>And the potential input features, that were quantified when taking biomarkers, would be:</p>
<ul class="simple">
<li><p>Globotriaosylceramide (Gb3)</p></li>
<li><p>Glucosylceramide</p></li>
<li><p>Galactosylceramide</p></li>
<li><p>Dermatan sulfate</p></li>
<li><p>Heparan sulfate</p></li>
<li><p>Hyaluronic acid</p></li>
<li><p>Glycogen</p></li>
<li><p>Free cholesterol</p></li>
<li><p>Calbindin D</p></li>
<li><p>Myostatin</p></li>
<li><p>Chitotriosidase</p></li>
<li><p>HbA1c</p></li>
<li><p>Glucose level</p></li>
<li><p>Lipofuscin [Molecular damage in aging Gladyshev <a class="reference external" href="http://et.al">et.al</a>]</p></li>
<li><p>Acidification [Molecular damage in aging Gladyshev <a class="reference external" href="http://et.al">et.al</a>]</p></li>
<li><p>enzymatic activity  [Molecular damage in aging Gladyshev <a class="reference external" href="http://et.al">et.al</a>]</p></li>
<li><p>Lysosomal-associated membrane protein expression [Molecular damage in aging Gladyshev <a class="reference external" href="http://et.al">et.al</a>]</p></li>
</ul>
<p>Alternative computational question for autophagy would be:</p>
<blockquote>
<div><p>What are some drugs that exhibit the same <a class="reference external" href="https://www.emcdda.europa.eu/document-library/computer-aided-silico-approaches-mode-action-analysis-and-safety-assessment-ostarine-and-4-methylamphetamine_en">mode of action</a> as example underneath.</p>
</div></blockquote>
<p>Screening huge libraries of compounds, like <a class="reference external" href="https://paperswithcode.com/dataset/zinc">ZINC</a> and after finding the same mode of action, perform <a class="reference external" href="https://medium.com/&#64;medicilon2004/effective-use-of-in-silico-tools-in-lead-optimization-62fd2cd3de55">lead optimization</a> might produce interesting candidates.</p>
<p><img alt="" src="https://nintil.com/images/2020-01-05-longevity/image-20200116074340112.png" /></p>
</section>
<section id="inflammation">
<h3>2. Inflammation<a class="headerlink" href="#inflammation" title="Permalink to this headline">#</a></h3>
<p><img alt="" src="https://www.mypathologyreport.ca/wp-content/uploads/2021/12/inflammatory-cells.jpg" /></p>
<p>Cellular senescence is a state in which cells stop dividing and enter a state of arrested growth. This is a natural protective mechanism that prevents damaged or potentially harmful cells from reproducing and potentially becoming cancerous. However, when a large number of senescent cells accumulate in the body, they can release harmful factors known as the senescence-associated secretory phenotype (SASP) that can lead to chronic inflammation and tissue damage.</p>
<p>Inflammation is a response to perceived threats in the body, such as pathogens or tissue damage. While this response is beneficial in the short term, chronic inflammation, or “inflammaging,” can cause tissue damage and increase the risk of developing diseases such as cancer. This type of inflammation can be triggered by factors such as the accumulation of senescent cells and unhealthy lifestyles, such as overeating.</p>
<p>Final interesting research question, that one could pursue, is elucidating the combined effect of most of these hallmarks. Askin the questions of the type</p>
</section>
</section>
<section id="aging-clocks-causality-and-the-future-of-evaluating-biological-age">
<h2>Aging clocks, causality and the future of evaluating biological age.<a class="headerlink" href="#aging-clocks-causality-and-the-future-of-evaluating-biological-age" title="Permalink to this headline">#</a></h2>
<p>Deep neural networks (DNNs) can be used to extract useful features from large sets of biomarkers available through common blood testing. For example, Putin et al. (2016) trained an ensemble of DNNs on biochemical parameters from 62,419 individuals and achieved impressive performance in the prediction of chronological age (R2 = 0.8). More interestingly, this approach allowed for feature extraction and identified albumin, glucose, alkaline phosphatase, urea, and erythrocytes as the most promising markers for predicting human chronological age (Putin et al., 2016). This exercise demonstrates the data mining potential of these new methods, even in well-investigated areas like biomarkers.</p>
<p>Feature selection is a critical and actionable area when considering the potential application of -omics sciences. While large-scale omics data are essential for providing insights into the aging phenotype, a selected number of actionable elements must be identified in order to enable intervention strategies. Galkin et al. (2020) also used deep learning (DL) to predict chronological age, but trained their algorithms on the taxonomic profiling of human gut microbiomes. This is a strategic choice, considering the growing importance of the microbiota and, in turn, nutrition for wellbeing and health.</p>
<p>One of the most important obstacles is waiting for human biology to confirm certain longevity intervention. What if we can simulate this?</p>
<p>Sure one potential solution are the animal models that we outline in chapter 4. But sometimes these animal models do not translate well to human outcomes. But can we do that computationally?</p>
<p>Aging clocks seem to predict true biological state (age). Great overview can be found <a class="reference external" href="https://www.technologyreview.com/2022/04/15/1050019/aging-clocks/">here</a>.</p>
<p>The most important aspect is the input data/biomarkers that are informative in infering the biological information.</p>
<ul class="simple">
<li><p><a class="reference external" href="https://epigenie.com/the-epigenetic-clock-from-deep-space-to-deep-learning/">The Epigenetic Clock</a></p></li>
<li><p><a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8279523/">DeepMAge: A Methylation Aging Clock Developed with Deep Learning</a></p></li>
<li><p><a class="reference external" href="https://www.cell.com/trends/pharmacological-sciences/fulltext/S0165-6147(19)30114-2">Deep Aging Clocks</a></p></li>
<li><p>etc</p></li>
</ul>
<p>It still remains to be seen wether this will be enough to infer goodnes of a certain intervention. It would be great if this could be used as a clinical target proxy to pass the FDA trials. Time will tell.</p>
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            kernelName: "python3",
            path: "./."
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

              </div>
              
            </main>
            <footer class="footer-article noprint">
                
    <!-- Previous / next buttons -->
<div class='prev-next-area'>
    <a class='left-prev' id="prev-link" href="Chapter%202%20-%20Biology%20basics%20and%20measuring%20biology.html" title="previous page">
        <i class="fas fa-angle-left"></i>
        <div class="prev-next-info">
            <p class="prev-next-subtitle">previous</p>
            <p class="prev-next-title">Chapter 2 - Biology basics and measuring biology</p>
        </div>
    </a>
    <a class='right-next' id="next-link" href="Chapter%204%20-%20Longevity%20interventions%20and%20models.html" title="next page">
    <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Chapter 4 - Longevity interventions and models</p>
    </div>
    <i class="fas fa-angle-right"></i>
    </a>
</div>
            </footer>
        </div>
    </div>
    <div class="footer-content row">
        <footer class="col footer"><p>
  
    By Noah Weber<br/>
  
      &copy; Copyright 2022.<br/>
</p>
        </footer>
    </div>
    
</div>


      </div>
    </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="_static/scripts/pydata-sphinx-theme.js?digest=1999514e3f237ded88cf"></script>


  </body>
</html>